### ISSUE TITLE
MultiplicativeNoise and GaussNoise output value range clipped to [0,1]

### ISSUE THREAD

**Issue by bleutner** [albumentations-team/albumentations#1054]
## 🐛 Bug

When MultiplicativeNoise or GaussNoise results in values > 1, the output is clipped to 1.

## To Reproduce

```python 
import numpy as np
import albumentations as A
print("A version:\n", A.__version__)

a = np.array([[255,240],[0.1, 1]])
transform = A.ReplayCompose([
    A.MultiplicativeNoise(always_apply = True)  
]) 

np.random.seed(42)
am=transform(image=a)
expected = a * am["replay"]["transforms"][0]["params"]["multiplier"]

print("Input:\n", a)
print("Transformed:\n", am["image"])
print("Expected:\n", expected)
```

```
A version:
 1.1.0
Input:
 [[2.55e+02 2.40e+02]
 [1.00e-01 1.00e+00]]
Transformed:
 [[1.         1.        ]
 [0.0974908  0.97490802]]
Expected:
 [[2.48601546e+02 2.33977926e+02]
 [9.74908024e-02 9.74908024e-01]]
```

## Expected behavior

Expected outputs > 1.

## Environment

 - Albumentations version: 1.1.0
 - Python version: 3.8.12
 - OS: Linux
 - How you installed albumentations:  pip

**Comment by BloodAxe** [comment]
TLDR: It's not bug, it's a feature (c)

At some point back in history, a clipping of float values to [0..1] looked like a decent idea for augmentations like random brightness/gamma/noise etc. So it became adopted and now exists for the sake of backward compatibility. If you utterly need this feature you can copy-paste augmentation to your code-base and remove @clipped decorator on apply funciton.

PS: Maybe some day we will add Unclipped versions of affected augmentations.

**Comment by bleutner** [comment]
ok, thanks for the info!
